docs/source/getting-started/installation.rst

```diff
@@ -544,11 +544,16 @@ Nebius
   nebius iam get-access-token > ~/.nebius/NEBIUS_IAM_TOKEN.txt
   nebius --format json iam whoami|jq -r '.user_profile.tenants[0].tenant_id' > ~/.nebius/NEBIUS_TENANT_ID.txt
 
-**Optional**: You can specify a preferable project ID, which will be used if a project ID is required in the designated region.
 
-.. code-block:: shell
+**Optional**: You can specify a preferable project ID and fabric in `~/.sky/config.yaml`
+
+.. code-block:: yaml
 
-  echo $NEBIUS_PROJECT_ID > ~/.nebius/NEBIUS_PROJECT_ID.txt
+  # ~/.sky/config.yaml
+  nebius:
+      eu-north1:
+        project_id: project-e00xxxxxxxxxxxx
+        fabric: fabric-2
```

Comment:
I think this should be moved to cloud-setup/cloud-permissions/nebius.rst. We can just have a link in the "Optional:" part to that page.